emails/models.py

```diff
@@ -363,3 +372,17 @@ def delete(self, *args, **kwargs):
         self.user_profile.num_address_deleted += 1
         self.user_profile.save()
         return super(DomainAddress, self).delete(*args, **kwargs)
+
+    @property
+    def full_address(self):
+        return '%s@%s.%s' % (
+            self.address, self.user_profile.subdomain, DEFAULT_DOMAIN
+        )
```

Comment:
The reply code needs a `full_address` to put in the `From:` header.

Reply:
We want to use the `self.domain` instead of the `DEFAULT_DOMAIN` when the code for moving the subdomain registration to mozmail.com instead of relay.firefox.com